internal/fnruntime/container.go

```
@@ -181,7 +193,8 @@ func (f *ContainerFn) prepareImage() error {
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
 	defer cancel()
 	cmd = exec.CommandContext(ctx, dockerBin, args...)
-	if output, err := cmd.CombinedOutput(); err != nil {
+	output, err = cmd.CombinedOutput()
+	if !foundImageInLocalCache && err != nil {
```

Adding a note why we ignore the error here will help useful.